{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   TypeError: __init__() takes 2 positional arguments but 3 were given\n", "test_name": "test_auth", "rerun": "0"}}, "suite_name": "tests/test_auth.py"}}}, "date": "September 13, 2022", "start_time": 1663094486.0250306, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "1"}